grails-bom/build.gradle

```diff
@@ -25,6 +25,7 @@ ext {
     // version name = versionNameOverride ?: artifactId-prefixes[0]
     dependenciesOverview = """\
         com.bertramlabs.plugins:asset-pipeline-grails::
+        com.bertramlabs.plugins:asset-pipeline-gradle::
```

Comment:
you are not using SCOTL format.  Shouldn't that be `com.bertramlabs.plugins:asset-pipeline:gradle,grails:` ? or is there a chance the version numbers won't match? I don't think they have ever been released independently.

Reply:
This is intentional. They have indeed been out of sync. There is for example no `4.4.0` version of `asset-pipeline-grails`

```console
mattias@mattias-z4-linux:~/test/g622app$ mcs com.bertramlabs.plugins:asset-pipeline-grails:4*
Searching for com.bertramlabs.plugins:asset-pipeline-grails:4*...
Found 7 results

  Coordinates                                           Last updated
  ===========                                           ============
  com.bertramlabs.plugins:asset-pipeline-grails:4.5.2   07 Nov 2024 at 19:38 (CET)
  com.bertramlabs.plugins:asset-pipeline-grails:4.5.1   25 Jul 2024 at 18:00 (CEST)
  com.bertramlabs.plugins:asset-pipeline-grails:4.5.0   25 Jul 2024 at 17:11 (CEST)
  com.bertramlabs.plugins:asset-pipeline-grails:4.3.0   01 May 2023 at 18:20 (CEST)
  com.bertramlabs.plugins:asset-pipeline-grails:4.2.0   11 Apr 2023 at 17:36 (CEST)
  com.bertramlabs.plugins:asset-pipeline-grails:4.1.0   03 Mar 2023 at 17:57 (CET)
  com.bertramlabs.plugins:asset-pipeline-grails:4.0.0   03 Nov 2022 at 16:29 (CET)

mattias@mattias-z4-linux:~/test/g622app$ mcs com.bertramlabs.plugins:asset-pipeline-gradle:4*
Searching for com.bertramlabs.plugins:asset-pipeline-gradle:4*...
Found 8 results

  Coordinates                                           Last updated
  ===========                                           ============
  com.bertramlabs.plugins:asset-pipeline-gradle:4.5.1   25 Jul 2024 at 18:00 (CEST)
  com.bertramlabs.plugins:asset-pipeline-gradle:4.5.0   25 Jul 2024 at 17:11 (CEST)
  com.bertramlabs.plugins:asset-pipeline-gradle:4.4.0   26 Sep 2023 at 22:24 (CEST)
  com.bertramlabs.plugins:asset-pipeline-gradle:4.3.0   01 May 2023 at 18:20 (CEST)
  com.bertramlabs.plugins:asset-pipeline-gradle:4.2.0   11 Apr 2023 at 17:36 (CEST)
  com.bertramlabs.plugins:asset-pipeline-gradle:4.1.0   03 Mar 2023 at 17:57 (CET)
  com.bertramlabs.plugins:asset-pipeline-gradle:4.0.0   03 Nov 2022 at 16:29 (CET)
  com.bertramlabs.plugins:asset-pipeline-gradle:4.5.2   07 Nov 2024 at 19:38 (CET)
```